inversion-api/src/main/java/io/inversion/cloud/model/Collection.java

```diff
@@ -169,6 +169,31 @@ public Collection withName(String name)
       return this;
    }
 
+   /**
+    * @param name
+    * @return true if the name or aliases patch
+    */
+   public boolean hasName(String nameOrAlias)
+   {
+      if (nameOrAlias == null)
+         return false;
+
+      if (nameOrAlias.equalsIgnoreCase(this.name))
+      {
+         return true;
+      }
+
+      for (String alias : this.aliases)
+      {
+         if (nameOrAlias.equalsIgnoreCase(alias))
```

Comment:
maybe better:

this.aliases
     .stream()
     .anyMatch(nameOrAlias::equalsIgnoreCase);